docs/UserGuide.md

```diff
@@ -148,6 +148,72 @@ Examples:
 * `client note add 2 t/pref prefers emails to calls`
 * `client note add 4 t/meeting need to slowly convince him to sign the contract`
 
+### Deleting client notes: `client note delete`
+
```

Comment:
Following the format for `client delete INDEX`, maybe add a short one line statement to explain what this command does?